sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -608,5 +608,318 @@ public async Task AnalyzeOperationAnalyzeSentimentWithOpinionMining()
 
             Assert.AreEqual(TextSentiment.Mixed, analyzeSentimentDocumentsResults[0].DocumentSentiment.Sentiment);
         }
+
+        #region Extract Summary
```

Comment:
Tried adding tests similar to what we have to other endpoints, but kept it in `AnalyzeOperationTests` since it's still part of it. If we have more actions in the future to put here, we can consider separating them in multiple files and putting them in a `AnalyzeOperation` folder.

Reply:
There are some tests that are actually part of the `AnalyzeOperation` endpoint as a whole, and not part of the `ExtractSummary` API (for example, AAD testing), so I haven't added tests for them here since they're already verified in the tests above.

There are a few scenarios in which we don't have tests for the Analyze Operation endpoint, though, such as sending a null ID, or a null input document. Should we create an issue to track those or are they not scenarios worth concern?

Reply:
>sending a null ID, or a null input document

These will be generic tests that apply to all endpoints in the library and not just to analyze. I don't remember if we have validation for both in our library but worth checking. Feel free to open an issue to look more into it

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/23161